packages/core/src/lib/lit-core.ts

```diff
@@ -535,9 +588,10 @@ export class LitCore {
       .catch((error: NodeErrorV3) => {
         logErrorWithRequestId(
           requestId,
-          `Something went wrong, internal id for request: lit_${requestId}. Please provide this identifier with any support requests. ${error?.message || error?.details
-            ? `Error is ${error.message} - ${error.details}`
-            : ''
+          `Something went wrong, internal id for request: lit_${requestId}. Please provide this identifier with any support requests. ${
```

Comment:
This is a "could not connect to nodes error right?  Should we put that in the message?  Or is this like, another unspecified error?  because I also see "could not connect to nodes" above so maybe this is the catch-all for an unknown error?  in that case, this is fine.

Reply:
Yes, this is emitted if there was an error response from `fetch` where we get a failure response back from the nodes. So if the nodes dont respond with a response in the `2xx` range this will be logged.

Reply:
Ok could we change the string to "Error connecting to nodes" instead of "Something went wrong"?  I just think it's clearer